client/signup/steps/domains/index.jsx

```diff
@@ -491,7 +620,69 @@ class DomainsStep extends Component {
 		return [ 'domain' ].includes( flowName );
 	};
 
+	async addDomain( suggestion ) {
+		const {
+			domain_name: domain,
+			product_slug: productSlug,
+			supports_privacy: supportsPrivacy,
+		} = suggestion;
+
+		let registration = domainRegistration( {
+			domain,
+			productSlug,
+			extra: { privacy_available: supportsPrivacy },
+		} );
+
+		if ( supportsPrivacy ) {
+			registration = updatePrivacyForDomain( registration, true );
+		}
+
+		try {
+			await this.props.shoppingCartManager.addProductsToCart( [ registration ] );
+		} catch {
+			return;
```

Comment:
What kind of errors is this dropping?

Reply:
I copied and it was like that, now I removed